### PR TITLE
fix(duplicate): Closing modal after duplicate action

### PR DESCRIPTION
**Pull request by pavlovicnemanja92**
SD-2526 and small css fix for SD-2419

### CODE REVIEW

File: client/app/scripts/superdesk-authoring/views/send-item.html [hunk 1]
```diff
@@ -50,17 +50,17 @@ <h3 translate ng-show="macros">Transformation</h3>
...
                 <li ng-show="mode !== 'authoring'">
                     <button class="send"
                         ng-click="send(true)"
                         ng-disabled="!selectedDesk || (mode !== 'ingest' && selectedStage._id === item.task.stage) || config"
                         translate>send and open</button>
                 </li>
+                <li ng-show="mode === 'authoring' && canSendAndContinue()">
+                    <button class="send" ng-click="sendAndContinue()"
+                    ng-disabled="!selectedDesk || (mode !== 'ingest' && selectedStage._id === item.task.stage)"
+                    translate>send and continue</button>
```
Comment (plamut) [inline client/app/scripts/superdesk-authoring/views/send-item.html]: Capital `s` here? Or is this done in CSS?

File: client/app/scripts/superdesk-archive/controllers/baseList.js [hunk 2]
```diff
@@ -28,6 +28,9 @@ define(['lodash'], function(_) {
                     $scope.selected.fetch = item;
                 });
             }
+            if (!$location.search().fetch) {
+                $scope.selected.fetch = null;
+            }
```
Comment (plamut) [inline client/app/scripts/superdesk-archive/controllers/baseList.js]: Test? :wink:

File: client/app/scripts/superdesk-authoring/views/send-item.html (hunk 1 above)
Comment (pavlovicnemanja92) [inline client/app/scripts/superdesk-authoring/views/send-item.html]: In all of this buttons, text transformation is done by css.

Comment (plamut) [inline client/app/scripts/superdesk-authoring/views/send-item.html]: OK, good then, never mind.

**Comment by mugurrus** [comment]
:+1:

File: client/app/scripts/superdesk-authoring/views/send-item.html (hunk 1 above)
Comment (petrjasek) [inline client/app/scripts/superdesk-authoring/views/send-item.html]: fyi it's better to do this in html, in e2e tests it's not consistent in chrome/firefox, chrome will give you uppercase and firefox will be lowercase

Comment (plamut) [inline client/app/scripts/superdesk-authoring/views/send-item.html]: Aha, good to know that, thanks!

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**